src/components/select/select.component.html

```diff
@@ -52,6 +53,7 @@
     class="ux-select-container"
     [class.disabled]="disabled"
     role="combobox"
+    [attr.aria-controls]="id + '-typeahead'"

```

Comment:
This attribute is now duplicated.
![image](https://user-images.githubusercontent.com/26114101/184098225-cd8c4841-5982-48d5-bb72-6d26212918dc.png)

Here's the practices guide for a combobox:
https://w3c.github.io/aria-practices/examples/combobox/combobox-autocomplete-list.html

So as you can see, the role  and other attributes need to be moved from here.